### PR TITLE
Blindness removes your ability to see colours

### PR DESCRIPTION
**Pull request by nightred**
## About The Pull Request

When blind you can make out a small amount of detail near you, but being able to see colour should not be part of that. You can make out the shapes of objects based on feeling around near you but you cannot tell that the door is red, or the folder is green.

## Why It's Good For The Game

Blind people can not make out color with their hands, now it is harder to function as a blind person.

## Changelog
:cl:
tweak: When blind you should not be able to make out colours, only the shapes near you.
/:cl:

### CODE REVIEW

**Comment by XDTM** [comment]
If you're making it a trait you could also add checks to the hypnotic stupor status effect

**Comment by nightred** [comment]
I have only made "monochromatic" have a trait that is trackable, I did not add or change any traits.
The only reason this was done was i did not want to remove the monochromatic filter if you are monochromatic.
I did not think it would be smart to add a second screen filter that duplicated what the monochromatic screen filter was already doing.
When i remove the client color "/datum/client_colour/monochrome" would it also be removing the sub color "/datum/client_colour/monochrome/trance"? I did not think it would.

File: code/modules/mob/status_procs.dm [hunk 1]
```diff
@@ -30,6 +30,8 @@
 			if(stat == CONSCIOUS || stat == SOFT_CRIT)
 				throw_alert("blind", /obj/screen/alert/blind)
 			overlay_fullscreen("blind", /obj/screen/fullscreen/blind)
+			// You are blind why should you be able to make out details like color, only shapes near you
+			src.add_client_colour(/datum/client_colour/monochrome/blind)
```
Comment (nemvar) [inline code/modules/mob/status_procs.dm]: Remove the vestigal src in front of the proc calls and this is good to go.

Comment (nightred) [reply code/modules/mob/status_procs.dm]: Should be fixed up now

**Comment by actioninja** [comment]
I'm working on a bigger blindness refactor at the moment but this is a solid tweak that I'll have to incorporate.

**Pull request merged**